### PR TITLE
Pull docs from aragonjs repo

### LINKED ISSUE #65: Duplicate docs of @aragon/wrapper
Currently we have the wrapper documented in both:
image
https://hack.aragon.org/docs/aragonjs-ref.html#wrapper
https://hack.aragon.org/docs/aragonjs-ref-clients.html

Solution A: 
- remove the "Reference documentation for client/wrappers" (although it seems to be the one that's more styled/mantained)

Solution  B:
- remove the Wrapper section from "Reference documentation"
- rename this section to "Reference documentation for wrappers" as calling this a client is very confusing

### PR DESCRIPTION
**Pull request by kernelwhisperer**
Added `scripts/sync-aragonjs-docs.js` which will download the `docs` from `aragon.js`.

What changed?
From the main "reference documentation" https://hack.aragon.org/docs/aragonjs-ref.html I removed the sections:
- "Wrapper"  (because it was duplicated, closes #65 )
- "Providers" (which now lives on it's own page) 
- "State" which is now called "Background Scripts"

See also https://github.com/aragon/aragon.js/pull/228

TODO:
- [x] change the script to download from `master` not `feat/generate-docs-from-sources` (once it's merged)

### CODE REVIEW

**Comment by cjyabraham** [comment]
These docs are not as well formatted as the current ones however we are in the process of improving the design so I have put it on my list to improve the css.

File: docs/js-guide-bg-scripts.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+id: aragonjs-guide-bg-scripts
+title: aragonAPI - Background Scripts guide
```
Comment (cjyabraham) [inline docs/js-guide-bg-scripts.md]: what do you think about removing the word "guide"?  It isn't used anywhere else in hack.aragon and I don't think it's needed.

File: docs/js-ref-app.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+// The default provider should be used in background scripts
+const backgroundScriptOfApp = new AragonApp()
+
+// The WindowMessage provider should be used for front-ends
+const frontendOfApp = new AragonApp(
+  new providers.WindowMessage(window.parent)
+)
+```
+
+### accounts
```
Comment (cjyabraham) [inline docs/js-ref-app.md:98]: is it easy to inject <hr> lines above each of these headings to help them stand out more as we've done [here](https://hack.aragon.org/docs/aragonjs-ref.html#events)?  I think once we implement the new design we'll want to remove them but if it's easy to put them in for a short-time then maybe we should as right now it's hard to read.

Comment (kernelwhisperer) [reply docs/js-ref-app.md:98]: Looks like it's only possible *after* each heading:
image

To style these docs would require a custom script ran after generating them :thinking:

Comment (cjyabraham) [reply docs/js-ref-app.md:98]: Ok.  I would say let's not worry about styling for now.  Within the next few weeks we'll be moving to the new aragon.org design and I'll make sure these get treated appropriately.

**Review by cjyabraham — APPROVED** [review]

**Pull request merged**